templates/web/task_detail.html

```diff
@@ -159,7 +159,7 @@ <h5 title="Top 5 plagiarism matches for some checked submit of this student" cla
                             <input type="file" name="solution" multiple hidden id="upload-choose"
                                    onclick="this.value=null;"
                                    onchange="this.form.submit()">
-                            <span class="btn btn-sm btn-primary" onclick="document.getElementById('upload-choose').click()">Upload</span>
+                            <button id="upload_button" class="btn btn-sm btn-primary" onclick="document.getElementById('upload-choose').click()" data-deadline="{{ deadline|date:'c' }}">Upload</button>
```

Comment:
Just using a button doesn't work, if you click on "Upload", it will send an empty request to the server :) So either the button behavior should be fixed, or we should keep `span` and disable it manually (by changing color and disabling the click functionality).